src/cmd/manage_nsfs.js

```diff
@@ -653,6 +664,15 @@ function get_access_keys(action, user_input) {
     return { access_keys, new_access_key };
 }
 
+/**
+ * set_bucker_owner gets bucket owner from cache by its id and sets bucket_owner name on the bucket data
+ * @param {object} bucket_data 
+ */
+async function set_bucker_owner(bucket_data) {
```

Comment:
A couple of questions related to the account_cache:
1. Now we should use the CLI with a working server in the background? (before that we didn't have to)
2. The cache it per endpoint, and could be many endpoints, how does it work?
3. What happens if the user updates the bucket owner? (`sudo node src/cmd/manage_nsfs bucket update --name <bucket name> --owner <another_account>`) will he see the previous one?
4. Why did you decide to use it in all the operations and not just in bucket list?